### DISCUSSION TITLE
Where to provide feedback on prototype models?

### DISCUSSION THREAD

**Discussion by aj-stein-gsa** [usnistgov/OSCAL#2091] [Q&A] [answered]
Hi @iMichaela, the FedRAMP Automation Team have been evaluating some of the prototype models and wish to provide feedback on them. What is the current procedure for feedback on prototype models? Do we create new issues in the usnistgov/OSCAL repo? Do we use usnistgov/OSCAL-DEFINE? Some other mechanism. Let me know and we will engage how you would like.

Thanks,
A.J.

**Comment by iMichaela** [comment, marked as answer]
@AJ - All community comments are captured in the issues #2012  and #2013 so the community can process them in one place. Please consider the discussions and decisions in these 2 issues as updates that are in the queue to be implemented. CNCF team is working on the examples captured in the shared responsibility model issue #2012 to further validate the model's usability for the highlighted scenario.

**Reply by aj-stein-gsa** [reply]
Excellent, thanks for the prompt reply.
